tests/components/homekit_controller/common.py

```diff
@@ -73,13 +73,14 @@ def add(self, accessory):
 class Helper:
     """Helper methods for interacting with HomeKit fakes."""
 
-    def __init__(self, hass, entity_id, pairing, accessory):
+    def __init__(self, hass, entity, pairing, accessory):
         """Create a helper for a given accessory/entity."""
         from homekit.model.services import ServicesTypes
         from homekit.model.characteristics import CharacteristicsTypes
 
         self.hass = hass
-        self.entity_id = entity_id
+        self.entity = entity
```

Comment:
We shouldn't store the entity. It doesn't seem to be used anyway.

We should use the core to test the entity via state machine and service registry, and not access the entity directly for tests.

Reply:
Yeah I thought why not as I’d gone to the trouble of finding it. Will ditch.